person.rb

```diff
@@ -0,0 +1,28 @@
+require_relative 'nameable'
+
+class Person < Nameable
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+    @rentals = []
+    super()
+  end
+  attr_accessor :name, :age
+  attr_reader :id, :rentals
+
+  def correct_name
+    @name
+  end
+
+  private
+
+  def of_age?
+    @age >= 18
+  end
+
+  def can_use_services?
+    of_age? || @parent_permission
+  end
+end
```

Comment:
- [ ] Please make sure that a method for adding rentals is present. The `person` itself should be sent as a parameter to create the new rental achieving the 'has-many' association. :point_down: 

  ```ruby
  def add_rental(book, date)
      Rental.new(date, book, self)
  end
  ```